fields/types/cloudinaryimage/CloudinaryImageField.js

```diff
@@ -292,7 +295,7 @@ module.exports = Field.create({
 		if (this.state.userSelectedFile || this.state.removeExisting) {
 			const value = this.state.userSelectedFile
 				? `upload:${this.state.uploadFieldPath}`
-				: '';
+				: `upload:remove`;
```

Comment:
@JedWatson Figured this out, it was failing validation, needs to be set to delete. See https://github.com/keystonejs/keystone/blob/master/fields/types/cloudinaryimage/CloudinaryImageType.js#L293